### ISSUE TITLE
Timing Diagram is not responsive on large screens

### ISSUE THREAD

**Issue by rexwangcc** [DataBiosphere/job-manager#565]
As a user, I'd expect the (beautiful) timing diagram could be responsive on large screens, right now it only uses a small portion of the panel:
![screen shot 2019-02-19 at 10 23 18 pm](https://user-images.githubusercontent.com/14366016/53064279-3d04b180-3495-11e9-844a-b3603fffbc3e.png)

**Comment by cjllanwarne** [comment]
Hey @rexwangcc could you define "responsive" here?

To me that would imply something like "interactive" but your report sounds like it's more like "not resizing to fill the whole screen width"?

**Comment by rexwangcc** [comment]
@cjllanwarne Yeah, I meant "not resizing to fill the whole screen width". I used `responsive` just because I was reading something like this: https://en.wikipedia.org/wiki/Responsive_web_design which made me thought `responsive` is the term to describe "web pages render well on a variety of devices and window or screen sizes" in the UI world. Since I'm not an expert on Front-end, I could be wrong :octocat:

**Comment by rsasch** [comment]
@rexwangcc As of `v0.5.8`, the graph width is dynamically determined when the browser page loads.  This means that it will be the right width when you navigate over to the "Timing Diagram" tab, but it will not re-size if you change the browser window size unless you reload the page.  Is this sufficient?
